test/pytorch_test_base.py

```diff
@@ -464,6 +464,11 @@
     'TestIndexingXLA': {
         'test_index_put_accumulate_large_tensor_xla',  # illegal memory access was encountered
     },
+
+    # test_metrics.py
+    'MetricsTest': {
+        'test_execute_time_metric',  # timing rigged for testing on CPU
```

Comment:
if the intent is only to run this on CPU, we need to add this test to `DISABLED_TORCH_TESTS_TPU_ONLY` too. 

However I thought this list only works for the test from the upstream. `test_metrics` is a test we defined and I think it won't go through this file. If you want to only want to run it on CPU, you can just check the device hardware type in the test.